grails-app/conf/Config.groovy

```diff
@@ -381,6 +380,123 @@ openboxes.dashboard.catalogsSummary.enabled = true
 // Default value for news summary
 openboxes.dashboard.newsSummary.newsItems = []
 
+openboxes {
```

Comment:
@jmiranda Should we have the same approach for Stock Movement endpoints? If I am understanding this correctly, once this will be merged to obgm branch there will be issues with additional /openboxes path at the beginning of url for this feature (because it's being added in apiClient currently).
As we discussed recently, we'd like to have a menu structure returned from api and the same could go for all endpoints we're using in SM, imho this could be refactored even as a part of OBPIH if you're ok with that.

Reply:
@awalkowiak Yeah I figured we'd have to rework this a bit for OGBM. I'm open to suggestions for how it should work. For now, as long as the current code doesn't break anything I'm fine with merging as-is.